packages/bot-web-ui/src/stores/load-modal-store.ts

```diff
@@ -427,12 +427,16 @@ export default class LoadModalStore implements ILoadModalStore {
                     readOnly: true,
                     scrollbars: true,
                 });
-                load_options.workspace = this.local_workspace;
+                const { active_tab } = this.root_store.dashboard;
```

Comment:
@rupato-deriv maybe syntax ES6 would be better?
```suggestion
                const { dashboard: { active_tab} } = this.root_store;
```

Reply:
done ✅